ui/js/actions/content.js

```diff
@@ -505,3 +505,36 @@ export function doPublish(params) {
     });
   };
 }
+
+export function doAbandonClaim(claimId, txid, nout) {
+  return function(dispatch, getState) {
+    const state = getState();
+
+    dispatch(doCloseModal());
+
+    dispatch({
+      type: types.ABANDON_CLAIM_STARTED,
+      data: {
+        claimId: claimId,
+        txid: txid,
+        nout: nout,
+      },
+    });
+
+    const success = dispatch({
+      type: types.ABANDON_CLAIM_SUCCEEDED,
+      data: {
+        claimId: claimId,
+        txid: txid,
+        nout: nout,
+      },
+    });
+
+    lbry
+      .claim_abandon({
+        txid: txid,
+        nout: nout,
+      })
+      .then(success);
```

Comment:
What if the call fails?

Reply:
We should at least alert an error if we don't want a whole event chain. As is, this will trigger the success event on a failure.

Reply:
Done.